globus_action_provider_tools/utils.py

```diff
@@ -1,11 +1,11 @@
+import datetime
 from os import urandom
 
-import arrow
 from base62 import encodebytes as base62
 
 
 def now_isoformat():
-    return str(arrow.utcnow())
+    str(datetime.datetime.now(datetime.timezone.utc).isoformat())
```

Comment:
The missing `return` statement suggests that this function might not be in use (at least, not during unit testing).

Reply:
Aha, I have successfully discovered the need for better unit test coverage/type annotations...